server/repository/src/migrations/v1_06_00/item_add_is_active.rs

```diff
@@ -4,7 +4,8 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
     sql!(
         connection,
         r#"
-        ALTER TABLE item ADD COLUMN is_active BOOLEAN NOT NULL DEFAULT TRUE;
+            ALTER TABLE item ADD COLUMN is_active BOOLEAN NOT NULL DEFAULT TRUE;
+            CREATE INDEX "index_item_is_active" ON "item" ("is_active");
```

Comment:
This is a new index isn't it?

Reply:
Yea I just saw it yesterday and wondered why not indexed - in theory soft delete booleans will be queried fairly frequently to filter the records out. Checked in team chat and agreed to do it.